lint/package.go

```diff
@@ -33,6 +33,7 @@ var (
 	falseValue = 2
 	notSet     = 3
 
+	go121 = goversion.Must(goversion.NewVersion("1.21"))
```

Comment:
I'm wondering: isn't go 1.21 already EOL? Because if it is, we should better use the existing go122 constant.

Reply:
> isn't go 1.21 already EOL?

Yes, go 1.21 is EOL.

> Because if it is, we should better use the existing go122 constant.

Does this mean "revive does not care about EOL go"?
Then, I think we should just add clear/max/min to `builtFunctions` unconditionaly.

However, I'm not unwilling to use `go122` / `IsAtLeastGo122` here.

Reply:
> Then, I think we should just add clear/max/min to builtFunctions unconditionaly.

👍

Suggesting to avoid using function named max/min is not really a problem, even for version under go 1.21. They could be renamed even if max/min is not available

Reply:
revive itself requires Go 1.21 and it will eventually require 1.22 because 1.21 is EOL but it should be possible to use revive to analyze Go code bases of any version.